src/loaders/index.ts

```diff
@@ -54,16 +54,22 @@ export default async function init(
 
   setContext({ userAgent: cliConfig.userAgent })
 
-  // cache location & ISP info
-  getPublicNetworkInfo()
-    .then(({ city, hostname, isp, privateIp, publicIp }) => {
-      log.info(
-        `Monika is running from: ${city} - ${isp} (${publicIp}) - ${hostname} (${privateIp})`
+  if (flags.verbose || isSymonMode) {
+    // cache location & ISP info
+    getPublicNetworkInfo()
+      .then(({ city, hostname, isp, privateIp, publicIp }) => {
+        log.info(
+          `Monika is running from: ${city} - ${isp} (${publicIp}) - ${hostname} (${privateIp})`
+        )
+      })
+      .catch((error) =>
+        log.warn(`Failed to obtain location/ISP info. Got: ${error}`)
       )
-    })
-    .catch((error) =>
-      log.warn(`Failed to obtain location/ISP info. Got: ${error}`)
-    )
+  } else {
+    // if note verbose, remove location details
+    ;`Monika is running from: City - isp (y.y.y.y) - localhost (x.x.x.x)`
```

Comment:
I think it should display "Monika is running" without any y.y.y.y or x.x.x.x as it is pretty unnecessary to me.

Reply:
Ok...i had wanted to preserve the original length, format and copywriting so that the notification channels keep their dimensions etc.